configs/LLNL-Sierra-IBM-power9-V100-Infiniband/caliper_cuda.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+variables:
+  modifier_package_name: 'caliper-cuda'
```

Comment:
Please remove from this PR, lets only have the platform independent modifier for now.